man/systemd.service.xml

```diff
@@ -1746,6 +1746,16 @@ WantedBy=multi-user.target</programlisting>
       <citerefentry><refentrytitle>systemd.kill</refentrytitle><manvolnum>5</manvolnum></citerefentry>
       for details on how you can influence the way systemd terminates
       the service.</para>
+
+      <para>To avoid code duplication, it is preferable to use
```

Comment:
Hmm, can we move this to `sd_notify` manpage please? systemd.service is already incredibly long, and it's directed at end users, while the sd_notify is a man page for developers.